.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+- '6'
```

Comment:
Yes. I've done this here because I noticed it's actually how the travis cli tool reformats it whenever it generates config for you, and it'd be nice to be consistent with that.